doc/introduction/data.rst

```diff
@@ -28,9 +28,19 @@ The :func:`~pennylane.data.load` function returns a ``list`` with the desired da
 
 >>> H2datasets = qml.data.load("qchem", molname="H2", basis="STO-3G", bondlength=1.1)
 >>> print(H2datasets)
-[<pennylane.data.dataset.Dataset object at 0x7f14e4369640>]
+[<Dataset = description: qchem/H2/STO-3G/1.1, attributes: ['molecule', 'hamiltonian', ...]>]
 >>> H2data = H2datasets[0]
 
+We can load multiple parameter values at once by providing a list of values instead of only a single value.
+To load all possible values, use the special keyword "full".
+
+>>> multiple_datasets = qml.data.load("qchem", molname="H2", basis="full", bondlength=[0.5, 1.1])
```

Comment:
```suggestion
>>> H2datasets = qml.data.load("qchem", molname="H2", basis="full", bondlength=[0.5, 1.1])
```

Reply:
I changed it to avoid confusion with the above one which later gets converted to H2data but I'm fine with putting it back. will do rn